codegenerator/cli/src/config_parsing/system_config.rs

```diff
@@ -267,49 +278,218 @@ impl SystemConfig {
+                // )
+                // .context("Failed to get path to ABI relative to the root of the project")?;
+                // let fuel_abi = FuelAbi::parse(abi_path).context(format!(
+                //     "Failed to parse ABI for the contract {}",
+                //     g_contract.name
+                // ))?;
+                let events = g_contract
+                    .config
+                    .events
+                    .iter()
+                    .cloned()
+                    .map(|e| {
+                        Event::from_evm_event_config(
+                            EventConfig {
+                                event: format!("{}()", e.name),
+                                name: None,
+                            },
+                            &None,
+                        )
+                    })
+                    .collect::<Result<Vec<_>>>()
+                    .context(format!(
+                        "Failed parsing abi types for events in global contract {}",
+                        g_contract.name,
+                    ))?;
+
+                let contract = Contract::new(
+                    g_contract.name.clone(),
+                    g_contract.config.handler.clone(),
+                    events,
+                    None,
+                )?;
+
+                //Check if contract exists
+                unique_hashmap::try_insert(&mut contracts, contract.name.clone(), contract)
+                    .context("Failed inserting globally defined contract")?;
+            }
+        }
+
+        for network in &fuel_config.networks {
+            for contract in network.contracts.clone() {
+                //Add values for local contract
+                match contract.config {
+                    Some(l_contract) => {
+                        // let abi_path = path_utils::get_config_path_relative_to_root(
+                        //     project_paths,
+                        //     PathBuf::from(&l_contract.abi_file_path),
+                        // )?;
+                        // let fuel_abi = FuelAbi::parse(abi_path).context(format!(
+                        //     "Failed to parse ABI for the contract {}",
+                        //     contract.name
+                        // ))?;
+
+                        let events = l_contract
+                            .events
+                            .iter()
+                            .cloned()
+                            .map(|e| {
+                                Event::from_evm_event_config(
+                                    EventConfig {
+                                        event: format!("{}()", e.name),
+                                        name: None,
+                                    },
+                                    &None,
+                                )
+                            })
+                            .collect::<Result<Vec<_>>>()?;
+
+                        // FIXME: Support Fuel for contract
```

Comment:
What do you think about using todo!() macros for unhandled cases? Not important but maybe helps use find known todos and can fail with helpful messages if we forget them etc.